images/manageiq-base-worker/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER ManageIQ https://manageiq.org
 LABEL name="manageiq-base-worker" \
       summary="ManageIQ Worker Image"
 
-COPY docker-assets/entrypoint /usr/local/bin
-COPY docker-assets/manageiq_liveness_check /usr/local/bin
+COPY assets/entrypoint /usr/local/bin
```

Comment:
container-assets?

Reply:
Since there is no other assets, I just shortened it... I can change to container-assets if that's preferred.

Reply:
Not sure... @bdunne ?

Reply:
Yeah, I prefer `container-assets`

Reply:
Ok, will change.